logging/src/main/java/life/qbic/logging/api/LogMessage.java

```diff
@@ -0,0 +1,17 @@
+package life.qbic.logging.api;
+
+/**
+ * Simple logging message record that holds basic information like:
+ * <ul>
+ *   <li>application</li>
+ *   <li>log level</li>
+ *   <li>message</li>
+ *   <li>throwable</li>
+ * </ul>
+ *
+ * @since 1.0.0
+ */
+public record LogMessage(String application, LogLevel logLevel, String message,
+                         Throwable throwable) {
+
```

Comment:
What is the throwable meant to represent? Is it the cause for the message, is it related to the message directly? Can there be messages without throwables?